zetta_utils/__init__.py

```diff
@@ -3,12 +3,17 @@
 import os
 import sys
 import warnings
+import multiprocessing
 
 from .log import get_logger
 
 # Set global multiprocessing threshold
 MULTIPROCESSING_NUM_TASKS_THRESHOLD = 128
 
+# Set start method to `forkserver`` if not set elsewhere
```

Comment:
Add a comment here saying that this is done because `fork` is not compatible with dependencies, and that this code will be irrelevant by python3.14 when forkserver is made the default?